recipes/typechecks/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Drop the `build.sh` and use pip:

https://conda-forge.org/docs/meta.html#use-pip